src/clj_puppetdb/vcr.clj

```diff
@@ -22,21 +21,29 @@
         (assoc response :content-type
                         {:mime-type (.getMimeType content-type)
                          :charset   (.getCharset content-type)})))))
+
 (defn body->string
   [response]
   "Turn response body into a string."
   (let [charset (util/response-charset response)]
-    (update-in response [:body] #(slurp (BufferedReader. (InputStreamReader. % charset))))))
+    (->> (-> (get response :body)
```

Comment:
I'm a broken record on this but I'd slurp and spit here to simplify the code. Ultimate VCR performance isn't a concern.